src/buildercore/cfngen.py

```diff
@@ -83,9 +83,17 @@ def build_context(pname, **more_context): # pylint: disable=too-many-locals
     # post-processing
     if 'rds' in context['project']['aws']:
         default_rds_dbname = slugify(stackname, separator="")
+
+        # used to give mysql a range of valid ip addresses to connect from
+        net, bits = context['project']['aws']['subnet-cidr'].split('/', 1)
+        # todo: determine mask based on bits
+        mask = '255.255.255.0'
```

Comment:
this is hardcoded, but should be dynamic depending on `bits`. I can't find a Python 2 standard library as `ipaddress` is in Python>=3.3. Therefore, I would use this nice and stable library: https://netaddr.readthedocs.io/en/latest/tutorial_01.html#representing-networks-and-subnets